### PR TITLE
feat: wiring different buttons and design review changes [VUE 1169, 1…

### PR DESCRIPTION
**Pull request by tiarascoleman**
…170, 1172, 1197]
Screen Shot 2022-08-10 at 6 41 06 PM

New page look with working buttons/checkbox!

### CODE REVIEW

File: src/components/Settings/SaveSearchItem.vue [hunk 1]
```diff
@@ -1,61 +1,111 @@
...
+						}
+					}
+				}`,
+				variables: {
+					id: this.savedSearch?.id,
+					savedSearch: { isAlert }
+				}
+			}).then(result => {
+				const emailAlertData = result?.data?.my?.updateSaveSearch;
+				console.log(emailAlertData);
```
Comment (dyersituations) [inline src/components/Settings/SaveSearchItem.vue:98]: Do we need to do anything with the result for this mutation or can this `.then` be removed?

File: src/components/Settings/SaveSearchItem.vue [hunk 2]
```diff
@@ -1,61 +1,111 @@
...
 import KvButton from '~/@kiva/kv-components/vue/KvButton';
 import KvCheckbox from '~/@kiva/kv-components/vue/KvCheckbox';
 
 export default {
 	name: 'SaveSearchItem',
 	data() {
 		return {
-			showAlerts: false,
+			showAlerts: this.savedSearch?.isAlert,
+			deleteSearch: () => []
```
Comment (mcstover) [inline src/components/Settings/SaveSearchItem.vue:50]: This line can be removed. It's not in use.

File: src/components/Settings/SaveSearchItem.vue (hunk 1 above)
Comment (mcstover) [inline src/components/Settings/SaveSearchItem.vue:98]: This console log can be removed.

File: src/pages/Settings/SavedSearchBeta.vue [hunk 3]
```diff
@@ -75,9 +77,12 @@ export default {
 			}).then(result => {
 				const savedSearchData = result?.data?.my?.savedSearches;
 				this.totalCount = savedSearchData?.totalCount ?? 0;
-				this.savedSearches = [...savedSearchData.values];
+				this.savedSearches = [...savedSearchData?.values];
 			});
-		}
+		},
+		deleteSavedSearch(payload) {
+			console.log(payload);
```
Comment (dyersituations) [inline src/pages/Settings/SavedSearchBeta.vue:84]: Is this for a future ticket? I don't see this method used anywhere.

File: src/pages/Settings/SavedSearchBeta.vue [hunk 4]
```diff
@@ -75,9 +77,12 @@ export default {
 			}).then(result => {
 				const savedSearchData = result?.data?.my?.savedSearches;
 				this.totalCount = savedSearchData?.totalCount ?? 0;
-				this.savedSearches = [...savedSearchData.values];
+				this.savedSearches = [...savedSearchData?.values];
 			});
-		}
+		},
+		deleteSavedSearch(payload) {
```
Comment (mcstover) [inline src/pages/Settings/SavedSearchBeta.vue:83]: This method block can be removed for now as it's not in use.

File: src/components/Settings/SaveSearchItem.vue (hunk 1 above)
Comment (mcstover) [reply src/components/Settings/SaveSearchItem.vue:98]: No, I don't believe we do at this time. Good suggestion!

File: src/pages/Settings/SavedSearchBeta.vue (hunk 3 above)
Comment (mcstover) [reply src/pages/Settings/SavedSearchBeta.vue:84]: This method block can be removed for now as it's not in use.

**Pull request merged**